<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
  "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
<head>
<meta name="generator" content="jemdoc, see http://jemdoc.jaboc.net/" />
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<link rel="stylesheet" href="jemdoc.css" type="text/css" />
<title>Short Bio</title>
</head>
<body>
<table summary="Table for page layout." id="tlayout">
<tr valign="top">
<td id="layout-menu">
<div class="menu-category">Mikel Hernaez</div>
<div class="menu-item"><a href="index.html" class="current">Home</a></div>
<div class="menu-item"><a href="bio.html">Bio</a></div>
<div class="menu-item"><a href="teaching.html">Teaching</a></div>
<div class="menu-category">Research</div>
<div class="menu-item"><a href="projects.html">Topics</a></div>
<div class="menu-item"><a href="group.html">Group</a></div>
<div class="menu-item"><a href="publications.html">Publications</a></div>
<div class="menu-item"><a href="https://github.com/ML4BM-Lab">Software</a></div>
<div class="menu-item"><a href="patents.html">Patents</a></div>
<div class="menu-item"><a href="past_projects.html">Past&nbsp;Projects</a></div>
<div class="menu-category">External Links</div>
<div class="menu-item"><a href="https://scholar.google.com/citations?user=-mh8RQ8AAAAJ&hl=en">Google&nbsp;Scholar&nbsp;Profile</a></div>
<div class="menu-item"><a href="https://orcid.org/0000-0003-0443-2305">ORCID</a></div>
<div class="menu-item"><a href="https://cima.cun.es/en/research/research-programs/computational-biology-research-programs">Corporate&nbsp;Group&nbsp;Website</a></div>
</td>
<td id="layout-content">
<div id="toptitle">
<h1>Short Bio</h1>
</div>
<p>Dr. Hernaez has had highly interdisciplinary research training in the last years. From training in Information Theory during his PhD (2009-2012), followed by his training in Computational Biology during his postdoc at Stanford University (2013-2016, funded by a Stanford Data Science Initiative fellowship); to his previous position as Director of Computational Genomics the Carl R. Woese Institute for Genomic Biology (IGB) at the University of Illinois (UIUC), USA; where he had ample experience working on biology-centered interdisciplinary projects. In 2020 he moved back to Spain to lead the Computational Biology Program at the Center for Applied Medical Research (CIMA) of the University of Navarra. These positions have allowed him to develop clear expertise on the application of machine learning to solve biological problems, as evidenced by his critical involvement in multiple projects published in the most prestigious peer reviewed journals such as Nature Machine Intelligence, Nature Communications or Cancer Research as corresponding author.
</p>
<p>Among other grants, Dr. Hernaez was awarded the prestigious Marie S. Curie Fellowship from the EU (2020) and the Ramon y Cajal Fellowship, Spain (2023, equivalent to a National Career Award) to develop machine learning models to study the regulatory dynamics of cancer progression and drug resistance, both at bulk and single-cell level. As an example, and further funded by the US DoD and the Spanish Ministry of Science and Innovation, his lab, in collaboration with Mayo Clinic, has recently developed several methods to elucidate mechanistic alterations in metastatic cells associated with drug response in metastatic prostate cancer. They were able to find (and validate in vivo models) driver genes whose regulatory program was rewired in non-responders, opening the doors to new treatments for this lethal cancer. Regarding drug resistance, in collaboration with Prof. Olivier Gevaert (Stanford School of Medicine), they have developed novel drug repurposing models via Graph Neural Networks, and uncovered potential new drugs and drug targets for pancreatic cancer. Dr. Hernaez is also actively engaged with Dr. Fernandez-Granda (NYU, Center for Data Science), Dr. Karunya Kandamilla (Univ. of Minnesota) and Dr. Krishna Kalari (Mayo Clinic) on an ongoing project funded by the US NIH (with Dr. Hernaez as co-PI) on foundational models for transcriptomic data deconvolution, with the goal of taking single cell resolution to the clinical practice.
</p>
<p>Additionally, Dr. Hernaez has proposed several compression methods for different types of genomic information, some of which have been adopted by the International Organization for Standardization (ISO), where Dr. Hernaez has co-led the new ISO standard for genomic information representation which awarded him the ISO Excellence Award in 2023. He holds two international patents that are currently incorporated in the ISO standard. He is also actively collaborating with Philips in this regard, with whom Dr. Hernaez has an ongoing industrial partnership.
</p>
<p>Finally, Dr. Hernaez is currently supervising 8 PhD students and 2 postdoctoral fellows, and he has a long track record of advising students. While at Stanford, UIUC and UNAV, he advised both master and undergraduate students, several interactions resulting in publications in top journals (Bioinformatics). In addition, he is currently co-chairing the Master in Data Science from the University of Navarra, where he is actively supervising master students. He has organized more than 10 workshops and special sessions on the top conferences on Computational Biology (ISMB; ECCB; Stanford Compression forum); and he is a reviewer in the top journals of Computational Biology (Nature Biotech., Nature Comms., Bioinformatics).
</p>
<div id="footer">
<div id="footer-text">
Page generated 2024-11-11 08:47:47 CET, by <a href="https://github.com/wsshin/jemdoc_mathjax" target="blank">jemdoc+MathJax</a>.
</div>
</div>
</td>
</tr>
</table>
</body>
</html>
