<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
  "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
<head>
<meta name="generator" content="jemdoc, see http://jemdoc.jaboc.net/" />
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<link rel="stylesheet" href="jemdoc.css" type="text/css" />
<title>Short Bio</title>
</head>
<body>
<table summary="Table for page layout." id="tlayout">
<tr valign="top">
<td id="layout-menu">
<div class="menu-category">Mikel Hernaez</div>
<div class="menu-item"><a href="index.html" class="current">Home</a></div>
<div class="menu-item"><a href="bio.html">Bio</a></div>
<div class="menu-item"><a href="cv/cv_mhernaez.pdf">Resume</a></div>
<div class="menu-item"><a href="teaching.html">Teaching</a></div>
<div class="menu-category">Research</div>
<div class="menu-item"><a href="projects.html">Projects</a></div>
<div class="menu-item"><a href="group.html">Group</a></div>
<div class="menu-item"><a href="publications.html">Publications</a></div>
<div class="menu-item"><a href="software.html">Software</a></div>
<div class="menu-item"><a href="patents.html">Patents</a></div>
<div class="menu-category">External Links</div>
<div class="menu-item"><a href="https://scholar.google.com/citations?user=-mh8RQ8AAAAJ&hl=en">Google&nbsp;Scholar&nbsp;Profile</a></div>
<div class="menu-item"><a href="https://github.com/mitogen">MITOGEN</a></div>
<div class="menu-item"><a href="https://compression.stanford.edu/">Stanford&nbsp;Compression&nbsp;Forum</a></div>
<div class="menu-item"><a href="https://mpeg.chiariglione.org/standards/mpeg-g">MPEG-G</a></div>
</td>
<td id="layout-content">
<div id="toptitle">
<h1>Short Bio</h1>
</div>
<p>Dr. Hernaez has had highly interdisciplinary research training in the last years. From training in Information Theory during his PhD (2009-2012), followed by his training in Computational Biology during his postdoc at Stanford University (2013-2016, funded by a Stanford Data Science Initiative fellowship); to his previous position as Director of Computational Genomics the Carl R. Woese Institute for Genomic Biology (IGB) at the University of Illinois (UIUC), USA; where he had ample experience working on biology-centered interdisciplinary projects. In 2020 he moved back to Spain to lead the Computational Biology Program at the Center for Applied Medical Research (CIMA), University of Navarra.</p>
<p>His research focuses on applying statistical learning and Bayesian methods to genomic information to learn useful patterns in the data. The learning of these patterns has led to the proposal of several compression methods for different types of genomic information, some of which have been adopted by the International Organization for Standardization (ISO), where he has co-led the new ISO standard for genomic information representation. In this context, Dr. Hernaez has been awarded several grants to bring next-generation sequencing to the clinical bedside through Compressive Learning, and holds two international patents on methods for genomic data representation that are currently incorporated in the upcoming ISO standard for genomic information representation. He is also actively collaborating with Philips on several projects in genomic data handling.</p>
<p>Dr. Hernaez was awarded in 2020 the prestigious Marie S. Curie Fellowship from the EU to develop machine learning methods to characterize the altered transcriptional dynamics associated with cancer progression. As an example, his lab has recently developed a method to elucidate mechanistic alterations in cells associated with drug response in prostate cancer. They were able to find (and validate in in vitro models) driver genes whose regulatory program was rewired in non-responders, opening the doors to new treatments of metastatic castration-resistant prostate cancer (in collaboration with Mayo Clinic). This work has resulted in several publications and is fostering important collaborations: the development of Bayesian methods for pan-cancer analysis of driver genes (in collaboration with Stanford University); the application of the developed GRNs at single-cell level to elucidate brain regions associated to aggressive behavior in honeybees (in collaboration with UIUC); and the development  machine learning models for deconvolving transcriptomic data (in collaboration with NYU, Center for Data Science and Univ. of Minnesota).</p>
<div id="footer">
<div id="footer-text">
Page generated 2022-02-21 12:44:07 CET, by <a href="http://jemdoc.jaboc.net/">jemdoc</a>.
</div>
</div>
</td>
</tr>
</table>
</body>
</html>
