<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
  "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
<head>
<meta name="generator" content="jemdoc, see http://jemdoc.jaboc.net/" />
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<link rel="stylesheet" href="jemdoc.css" type="text/css" />
<title>Relevant publications by topic</title>
</head>
<body>
<table summary="Table for page layout." id="tlayout">
<tr valign="top">
<td id="layout-menu">
<div class="menu-category">Mikel Hernaez</div>
<div class="menu-item"><a href="index.html" class="current">Home</a></div>
<div class="menu-item"><a href="bio.html">Bio</a></div>
<div class="menu-item"><a href="teaching.html">Teaching</a></div>
<div class="menu-category">Research</div>
<div class="menu-item"><a href="projects.html">Topics</a></div>
<div class="menu-item"><a href="group.html">Group</a></div>
<div class="menu-item"><a href="publications.html">Publications</a></div>
<div class="menu-item"><a href="https://github.com/ML4BM-Lab">Software</a></div>
<div class="menu-item"><a href="patents.html">Patents</a></div>
<div class="menu-item"><a href="past_projects.html">Past&nbsp;Projects</a></div>
<div class="menu-category">External Links</div>
<div class="menu-item"><a href="https://scholar.google.com/citations?user=-mh8RQ8AAAAJ&hl=en">Google&nbsp;Scholar&nbsp;Profile</a></div>
<div class="menu-item"><a href="https://orcid.org/0000-0003-0443-2305">ORCID</a></div>
<div class="menu-item"><a href="https://cima.cun.es/en/research/research-programs/computational-biology-research-programs">Corporate&nbsp;Group&nbsp;Website</a></div>
</td>
<td id="layout-content">
<div id="toptitle">
<h1>Relevant publications by topic</h1>
</div>
<ul>
<li><p>(*): Corresponding author
</p>
</li>
<li><p>(†): (Co-)First Author
</p>
</li>
</ul>
<h2>Pre Prints</h2>
<ul>
<li><p><a href="https://arxiv.org/abs/2311.11991" target=&ldquo;blank&rdquo;>Sweetwater: An interpretable and adaptive autoencoder for efficient tissue deconvolution</a><br />
J. de la Fuente†, N. Legarra†, G. Serrano, I. Marin-Goni, A. Diaz-Mazkiaran, M. Benito, A. Garcia-Osta, K. R Kalari, C. Fernandez-Granda, I. Ochoa*, <b>M. Hernaez*</b><br />
<i>arXiv</i>, 2024
</p>
</li>
</ul>
<h2>Novel insights into the (patho)physiology of diseases via Machine Learning</h2>
<ul>
<li><p><a href="https://www.nature.com/articles/s41467-024-54005-7" target=&ldquo;blank&rdquo;>Uncovering functional lncRNAs by scRNA-seq with ELATUS</a><br />
E. Goñi, A. M. Mas , J. Gonzalez, [&hellip;] , Maite Huarte*, <b>M. Hernaez*</b><br />
<i>Nature Communications</i>, 2024 
</p>
</li>
</ul>
<ul>
<li><p><a href="https://www.nature.com/articles/s41467-024-49529-x" target=&ldquo;blank&rdquo;>Single-cell transcriptional profile of CD34+ hematopoietic progenitor cells from del(5q) Myelodysplastic Syndromes and impact of lenalidomide</a><br />
G. Serrano†, N. Berastegi†, A. Diaz-Mazkiaran†, [&hellip;], F. Prosper*, T. Ezponda*, <b>M. Hernaez*</b><br />
<i>Nature Communications</i>, 2024
</p>
</li>
</ul>
<ul>
<li><p><a href="https://aacrjournals.org/cancerres/article/83/8/1361/725101/Bayesian-Machine-Learning-Enables-Identification" target=&ldquo;blank&rdquo;>Identification of transcriptional network disruptions in drug-resistant metastatic prostate cancer with TraRe</a><br />
C. Blatti†, J. de la Fuente†, H. Gao, I. Marin, Z. Chen, S. D. Zhao, W. Tan, R. Weinshilbaum, K. R. Kalari, L. Wang*, <b>M. Hernaez*</b><br />
<i>Cancer Research</i>, 2023<br /> 
</p>
</li>
</ul>
<ul>
<li><p><a href="https://www.frontiersin.org/journals/immunology/articles/10.3389/fimmu.2023.1270843/full" target=&ldquo;blank&rdquo;>Optimization of a universal allogeneic CAR-T cells combining CRISPR and transposon-based technologies for treatment of Acute Myeloid Leukemia</a><br />
C. Calviño†, C. Ceballos†, A. Alfonso†, P. Jauregui, [&hellip;], <b>M. Hernaez*</b>, F. Prosper*, J. R. Rodriguez-Madoz*<br />
<i>Frontiers in Immunology</i>, 2023
</p>
</li>
</ul>
<ul>
<li><p><a href="https://www.science.org/doi/10.1126/sciadv.abo0514" target=&ldquo;blank&rdquo;>CAR Density Influences Antitumoral Efficacy of BCMA CAR T cells and Correlates with Clinical Outcome</a><br />
P. Rodriguez-Marquez†,  M. E. Calleja-Cervantes†,  G. Serrano†, [&hellip;], <b>M. Hernaez*</b>,  J. R. Rodriguez-Madoz*, F. Prosper*<br />
<i>Science Advances</i>, 2022 (Oral Presentation at ASH 2022)<br />
</p>
</li>
</ul>
<ul>
<li><p><a href="https://genome.cshlp.org/content/early/2021/03/01/gr.267013.120" target=&ldquo;blank&rdquo;>Cellular plasticity balances the metabolic and proliferation dynamics of a regenerating liver</a> <br />
U. V. Chembazhi, S. Bangru, <b>M. Hernaez</b>, A. Kalsotra<br />
<i>Genome Research</i>, 2021
</p>
</li>
</ul>
<h2>(Graph) Representation Learning in biomedicine</h2>
<ul>
<li><p><a href="https://openreview.net/pdf?id=3Fgylj4uqL" target=&ldquo;blank&rdquo;>Interpretable Causal Representation Learning for Biological Data in the Pathway Space</a><br />
J. de la Fuente, R. Lehmann, C. Ruiz-Arenas, [&hellip;], V. Lagani*, <b>M. Hernaez*</b><br />
<i>International Conference on Learning Representations (ICLR)</i>, 2025
</p>
</li>
</ul>
<ul>
<li><p><a href="https://arxiv.org/pdf/2311.12670" target=&ldquo;blank&rdquo;>Towards a more inductive world for drug repurposing approaches</a><br />
J. de la Fuente†, G. Serrano†, U. Veleiro†, [&hellip;], S. Vicent, O. Gevaert*, <b>M. Hernaez*</b><br />
<i>Nature Machine Intelligence</i>, 2025 (Oral Presentation at Neurips - AI4D3 Workshop, 2024)<br />
</p>
</li>
</ul>
<ul>
<li><p><a href="https://academic.oup.com/nar/article/52/9/e44/7643294" target=&ldquo;blank&rdquo;>NetActivity enhances transcriptional signals by combining gene expression into robust gene set activity scores through interpretable autoencoders</a><br />
C. Ruiz-Arenas, I. Marin, L. Wang, I. Ochoa, J.L. Perez-Jurado*, <b>M. Hernaez*</b><br />
<i>Nucleic Acids Research</i>, 2024 (Oral Presentation a ISMB 2022)<br />
</p>
</li>
</ul>
<ul>
<li><p><a href="https://academic.oup.com/bioinformatics/article/40/1/btad774/7491592" target=&ldquo;blank&rdquo;>GeNNius: An ultrafast drug-target interaction inference method based on graph neural networks</a><br />
U. Veleiro, J. De la Fuente, G. Serrano, [&hellip;], I. Ochoa, G. Gevaert*, <b>M. Hernaez*</b><br />
<i>Bioinformatics</i>, 2024<br />
</p>
</li>
</ul>
<ul>
<li><p><a href="https://academic.oup.com/bioinformatics/article/38/9/2488/6543609" target=&ldquo;blank&rdquo;>JIND: Joint Integration and Discrimination for Automated Single-Cell Annotation</a><br />
M. Goyal, G. Serrano, J. Argemi, I. Shomorony, <b>M. Hernaez*</b>, I. Ochoa*<br />
<i>Bioinformatics</i>, 2022<br /> (Oral Presentation a MLBM Workshop 2021)<br />
</p>
</li>
</ul>
<ul>
<li><p><a href="https://www.nature.com/articles/s42003-022-03319-7" target=&ldquo;blank&rdquo;>SimiC enables the inference of complex gene regulatory dynamics across cell phenotypes</a><br />
J. Peng†, G. Serrano†, I. M. Traniello, M. E. Calleja-Cervantes, [&hellip;], I. Ochoa*, <b>M. Hernaez*</b><br />
<i>Communications Biology</i>, 2022<br />
</p>
</li>
</ul>
<ul>
<li><p><a href="https://academic.oup.com/bioinformatics/article/36/2/558/5530163" target=&ldquo;blank&rdquo;>Comparison of single gene and module-based methods for modeling gene regulatory networks</a><br />
<b>M. Hernaez*</b>, C. Blatti, O. Gevaert*<br />
<i>Bioinformatics</i>, 2020<br />
</p>
</li>
</ul>
<h2>Genomic information representation</h2>
<ul>
<li><p><a href="https://doi.org/10.1038/s42003-024-06249-8" target=&ldquo;blank&rdquo;>Genie: the first open-source ISO/IEC encoder for genomic data</a><br />
F. Muntefering, Y. Adhisantoso, S. Chandak, J. Oestermann, I. Ochoa*, <b>M. Hernaez*</b><br />
<i>Communications Biology</i>, 2024<br />     
</p>
</li>
</ul>
<ul>
<li><p><a href="https://ieeexplore.ieee.org/abstract/document/9455132" target=&ldquo;blank&rdquo;>An introduction to MPEG-G: The first open ISO/IEC standard for the compression and exchange of genomic sequencing data</a> <br />
J. Voges, <b>M. Hernaez</b>, M. Mattavelli, J. Ostermann<br />
<i>Proceedings of the IEEE</i>, 2021<br />
</p>
</li>
</ul>
<ul>
<li><p><a href="https://www.nature.com/articles/s43588-021-00089-w" target=&ldquo;blank&rdquo;>Towards scalable genomic data access</a><br />
<b>M. Hernaez</b> <br />
<i>Nature Computational Science</i>, 2021<br />
</p>
</li>
</ul>
<ul>
<li><p><a href="https://academic.oup.com/bioinformatics/article/36/7/2275/5674036" target=&ldquo;blank&rdquo;>GABAC: an arithmetic coding solution for genomic data</a> <br />
J. Voges, T. Paridaens, F. Müntefering, L. S. Mainzer, B. Bliss, M. Yang, I. Ochoa, J. Fostier, J. Ostermann*, <b>M. Hernaez*</b><br />
<i>Bioinformatics</i>, 2020 <br />
</p>
</li>
</ul>
<ul>
<li><p><a href="https://academic.oup.com/bioinformatics/article/36/18/4810/5865850" target=&ldquo;blank&rdquo;>GPress: a framework for querying general feature format (GFF) files and expression files in a compressed form</a> <br />
Q. Meng, I. Ochoa<b>, </b>M. Hernaez**<br />
<i>Bioinformatics</i>, 2020 <br />
</p>
</li>
</ul>
<ul>
<li><p><a href="https://academic.oup.com/bioinformatics/article/36/7/2275/5674036" target=&ldquo;blank&rdquo;>GABAC: an arithmetic coding solution for genomic data</a> <br />
J. Voges, T. Paridaens, F. Müntefering, L. S. Mainzer, B. Bliss, M. Yang, I. Ochoa, J. Fostier, J. Ostermann, <b>M. Hernaez*</b><br />
<i>Bioinformatics</i>, 2020 <br />
</p>
</li>
</ul>
<ul>
<li><p><a href="https://www.annualreviews.org/doi/10.1146/annurev-biodatasci-072018-021229" target=&ldquo;blank&rdquo;>Genomic data compression</a> <br />
<b>M. Hernaez*</b>, D. Pavlichin, T. Weissman, I. Ochoa* <br />
<i>Annual Review of Biomedical Data Science</i>, 2019 <br />
</p>
</li>
</ul>
<ul>
<li><p><a href="https://www.nature.com/articles/s41598-019-51418-z" target=&ldquo;blank&rdquo;>Denoising of Aligned Genomic Data</a> <br />
I. fischer-Hwang, I. Ochoa, T. Weissman, <b>M. Hernaez*</b><br />
<i>Scientific Reports</i>, 2019<br />
</p>
</li>
</ul>
<ul>
<li><p><a href="https://ieeexplore.ieee.org/document/8712770" target=&ldquo;blank&rdquo;>AliCo:  a new efficient representation for SAM files</a> <br />
I. Ochoa, H. Li, F. Baumgarte, C. Hergenrother, J. Voges and  <b>M. Hernaez*</b>
<i>Data Compression Conference (DCC)</i>, 2019<br />
</p>
</li>
</ul>
<ul>
<li><p><a href="https://doi.org/10.1093/bioinformatics/bty1015" target=&ldquo;blank&rdquo;>SPRING: A next-generation compressor for FASTQ data</a> <br />
S. Chandak, K. Tatwawadi, I. Ochoa,  <b>M. Hernaez*</b> and T. Weissman
<i>Bioinformatics, 2018</i><br />
</p>
</li>
</ul>
<ul>
<li><p><a href="https://doi.org/10.1093/bioinformatics/btx737" target=&ldquo;blank&rdquo;>CALQ: compression of quality values of aligned sequencing data</a><br />
J. Voges, J. Ostermann and <b>M. Hernaez*</b><br />
<i>Bioinformatics</i>, 2017<br />
</p>
</li>
</ul>
<ul>
<li><p><a href="http://ieeexplore.ieee.org/document/7923646/" target=&ldquo;blank&rdquo;>GeneComp, a new reference-based compressor for SAM files</a> <br />
R. Long, <b>M. Hernaez*</b>, I. Ochoa and T. Weissman<br />
<i>Data Compression Conference (DCC)</i>, 2017<br />
</p>
</li>
</ul>
<ul>
<li><p><a href="http://bib.oxfordjournals.org/content/early/2016/03/08/bib.bbw011" target=&ldquo;blank&rdquo;>Effect of lossy compression of quality scores on variant calling</a> <br />
I. Ochoa, <b>M. Hernaez*</b>, R. Goldfeder, T. Weissman and E. Ashley<br />
<i>Briefings in Bioinformatics</i>, 2016<br />
</p>
</li>
</ul>
<ul>
<li><p><a href="http://ieeexplore.ieee.org/document/7786169/" target=&ldquo;blank&rdquo;>Denoising of Quality Scores for Boosted Inference and Reduced Storage</a> <br />
I. Ochoa, <b>M. Hernaez*</b>, R. Goldfeder, T. Weissman and E. Ashley<br />
<i>Data Compression Conference (DCC)</i>, 2016<br />
</p>
</li>
</ul>
<ul>
<li><p><a href="http://ieeexplore.ieee.org/document/7786170/" target=&ldquo;blank&rdquo;>A cluster-based approach to compression of Quality Scores</a><br /> 
<b>M. Hernaez</b> , I. Ochoa and T. Weissman<br />
<i>Data Compression Conference (DCC)</i>, 2016<br />
</p>
</li>
</ul>
<ul>
<li><p><a href="http://ieeexplore.ieee.org/document/7786166/" target=&ldquo;blank&rdquo;>A 
An Evaluation Framework for Lossy Compression of Genome Sequencing Quality Values</a><br /> 
C. Alberti, N. Daniels, <b>M. Hernaez*</b>, J. Voges, R. L. Goldfeder, A. A. Hernandez-Lopez, M. Mattavelli, B. Berger<br />
<i>Data Compression Conference (DCC)</i>, 2016<br />
</p>
</li>
</ul>
<ul>
<li><p><a href="http://bioinformatics.oxfordjournals.org/content/31/19/3122" target=&ldquo;blank&rdquo;>QVZ: lossy compression of quality values</a> <br />
G. Malysa, <b>M. Hernaez*</b>, I. Ochoa, M. Rao, K. Ganesan and T. Weissman<br />
<i>Bioinformatics</i>, 2015<br />
</p>
</li>
</ul>
<ul>
<li><p><a href="http://bioinformatics.oxfordjournals.org/content/31/5/626.long" target=&ldquo;blank&rdquo;>iDoComp: a compression scheme for assembled genomes</a> <br />
I. Ochoa, <b>M. Hernaez*</b> and T. Weissman<br />
<i>Bioinformatics</i>, 2014<br />
</p>
</li>
</ul>
<ul>
<li><p><a href="https://www.ncbi.nlm.nih.gov/pubmed/25395305" target=&ldquo;blank&rdquo;>Aligned genomic data compression via improved modeling</a> <br />
I. Ochoa, <b>M. Hernaez*</b> and T. Weissman<br />
<i>Journal of bioinformatics and computational biology</i>, 2014 (presented at the GIW ISCB-Asia, 2014)<br />
</p>
</li>
</ul>
<h2>Network Information Theory (PhD Thesis)</h2>
<ul>
<li><p><a href="http://ieeexplore.ieee.org/document/6330081/" target=&ldquo;blank&rdquo;>On the Design of a Novel Joint Network-Channel Coding Scheme for the Multiple Access Relay Channel</a> <br />
<b>M. Hernaez</b>, P. M. Crespo and J. Del Ser<br />
<i>IEEE Journal on Selected Areas in Communications</i>, 2013<br />
</p>
</li>
</ul>
<ul>
<li><p><a href="http://ieeexplore.ieee.org/document/6692030/" target=&ldquo;blank&rdquo;>A Decode-and-Forward Scheme for Multihop Wireless Networks</a> <br />
<b>M. Hernaez</b>, P. M. Crespo and J. Del Ser<br />
<i>IEEE 78rd Vehicular Technology Conference (VTC Spring)</i>, 2013<br />
</p>
</li>
</ul>
<ul>
<li><p><a href="http://ieeexplore.ieee.org/document/6259797/" target=&ldquo;blank&rdquo;>A Flexible Channel Coding Approach for Short-Length Codewords</a> <br />
<b>M. Hernaez</b>, P. M. Crespo and J. Del Ser<br />
<i>IEEE Communications Letters</i>, 2012<br />
</p>
</li>
</ul>
<ul>
<li><p><a href="http://ieeexplore.ieee.org/document/5956446/" target=&ldquo;blank&rdquo;>A Novel Scheme for Message-Forwarding in Multi-Hop Ad-Hoc Wireless Networks</a> <br />
<b>M. Hernaez</b> and P. M. Crespo<br />
<i>IEEE 73rd Vehicular Technology Conference (VTC Spring)</i>, 2011<br />
</p>
</li>
</ul>
<ul>
<li><p><a href="http://ieeexplore.ieee.org/document/5956367/" target=&ldquo;blank&rdquo;>Joint Non-Binary LDPC-BICM and Network Coding with Iterative Decoding for the Multiple Access Relay Channel</a> <br />
<b>M. Hernaez</b>, P. M. Crespo and J. Del Ser<br />
<i>IEEE 73rd Vehicular Technology Conference (VTC Spring)</i>, 2011<br />
</p>
</li>
</ul>
<ul>
<li><p><a href="http://ieeexplore.ieee.org/document/5288464/" target=&ldquo;blank&rdquo;>Serially-Concatenated LDGM Codes for Correlated Sources over Gaussian Broadcast Channels</a> <br />
<b>M. Hernaez</b>, P. M. Crespo, J. Del Ser and J. Garcia-Frias<br />
<i>IEEE Communications Letters</i>, 2009<br />
</p>
</li>
</ul>
<h2>PhD Thesis</h2>
<ul>
<li><p><a href="Thesis_Mikel_Hernaez.pdf" target=&ldquo;blank&rdquo;>Joint Network-Channel Coding Schemes for Relay Networks</a>
</p>
</li>
</ul>
<div id="footer">
<div id="footer-text">
Page generated 2025-03-06 18:51:19 CET, by <a href="https://github.com/wsshin/jemdoc_mathjax" target="blank">jemdoc+MathJax</a>.
</div>
</div>
</td>
</tr>
</table>
</body>
</html>
