%% start of file `template.tex'.
%% Copyright 2006-2015 Xavier Danaux (xdanaux@gmail.com).
%
% This work may be distributed and/or modified under the
% conditions of the LaTeX Project Public License version 1.3c,
% available at http://www.latex-project.org/lppl/.


\documentclass[11pt,a4paper,sans]{moderncv}        % possible options include font size ('10pt', '11pt' and '12pt'), paper size ('a4paper', 'letterpaper', 'a5paper', 'legalpaper', 'executivepaper' and 'landscape') and font family ('sans' and 'roman')

%\usepackage[scale]{tgheros}
\usepackage{amsmath}

% moderncv themes
\moderncvstyle[center]{CLASSIC}
%\moderncvstyle{banking}                             % style options are 'casual' (default), 'classic', 'banking', 'oldstyle' and 'fancy'
\moderncvcolor{black}                               % color options 'black', 'blue' (default), 'burgundy', 'green', 'grey', 'orange', 'purple' and 'red'

%\renewcommand{\familydefault}{\sfdefault}         % to set the default font; use '\sfdefault' for the default sans serif font, '\rmdefault' for the default roman one, or any tex font name
%\nopagenumbers{}                                  % uncomment to suppress automatic page numbering for CVs longer than one page

% character encoding
%\usepackage[utf8]{inputenc}                       % if you are not using xelatex ou lualatex, replace by the encoding you are using
%\usepackage{CJKutf8}                              % if you need to use CJK to typeset your resume in Chinese, Japanese or Korean

% adjust the page margins
\usepackage[scale=0.75]{geometry}
%\setlength{\hintscolumnwidth}{3cm}                % if you want to change the width of the column with the dates
%\setlength{\makecvtitlenamewidth}{10cm}           % for the 'classic' style, if you want to force the width allocated to your name and avoid line breaks. be careful though, the length is normally calculated to avoid any overlap with your personal info; use this at your own typographical risks...

% personal data
\name{Mikel}{Hernaez}
%\title{Director of Computational Genomics, }                               % optional, remove / comment the line if not wanted
\address{CIMA University of Navarra, Pio XII, 55, 31008, Pamplona, Spain}%{postcode city}{country}% optional, remove / comment the line if not wanted; the "postcode city" and "country" arguments can be omitted or provided empty
%\phone[fixed]{+1~(650)~739~5404}                   % optional, remove / comment the line if not wanted; the optional "type" of the phone can be "mobile" (default), "fixed" or "fax"
\phone[mobile]{+34~(686)~872~682}
%\phone[fax]{+3~(456)~789~012}
\email{mhernaez@unav.es}                               % optional, remove / comment the line if not wanted
%\homepage{www.stanford.edu/~mhernaez}                         % optional, remove / comment the line if not wanted
%\social[linkedin]{john.doe}                        % optional, remove / comment the line if not wanted
%\social[twitter]{jdoe}                             % optional, remove / comment the line if not wanted
\social[github]{mikelhernaez}                              % optional, remove / comment the line if not wanted
%\extrainfo{additional information}                 % optional, remove / comment the line if not wanted
%\photo[64pt][0.4pt]{picture}                       % optional, remove / comment the line if not wanted; '64pt' is the height the picture must be resized to, 0.4pt is the thickness of the frame around it (put it to 0pt for no frame) and 'picture' is the name of the picture file
%\quote{Vision, mission, do website, clarify my topic}                                 % optional, remove / comment the line if not wanted

% bibliography adjustements (only useful if you make citations in your resume, or print a list of publications using BibTeX)
%   to show numerical labels in the bibliography (default is to show no labels)
\makeatletter\renewcommand*{\bibliographyitemlabel}{\@biblabel{\arabic{enumiv}}}\makeatother
%   to redefine the bibliography heading string ("Publications")
%\renewcommand{\refname}{Articles}

% bibliography with mutiple entries
%\usepackage{multibib}
%\newcites{book,misc}{{Books},{Others}}
%----------------------------------------------------------------------------------
%            content
%----------------------------------------------------------------------------------
\begin{document}
%\begin{CJK*}{UTF8}{gbsn}                          % to typeset your resume in Chinese using CJK
%-----       resume       ---------------------------------------------------------
\makecvtitle
\vspace{-25pt}
\vspace{-5pt}

\section{Current Positions}
\cventry{}{Director, Computational Biology and Translational Genomics Program}{Center for Applied Medical Research (CIMA) University of Navarra}{ SPAIN }{}{Principal Investigator, Machine Learning for Biomedicine Lab}
\cventry{}{Adjunct Professor}{Department of Physics and Applied Mathematics}{University of Navarra,  SPAIN}{}{}
\cventry{}{Associate Researcher}{Institute of Data Science and Artificial Intelligence (DATAI)}{University of Navarra, SPAIN}{}{}
\cventry{}{Principal Investigator}{Navarra Institute for Health Research (IdiSNA)}{SPAIN}{}{}

%\section{Current Projects and Research}
%
%\vspace{10pt}
%\cventry{}{Funded by the Marie Curie Fellowship from the European Comission}{Elucidating transcriptional rewiring on hematological malignancies via computational methods}{}{}{} 
%\begin{itemize}
%\item Development of computational methods to uncover dynamically-regulated GRNs.
%\item Elucidate transcriptional rewiring on hematological malignancies.
%\end{itemize}
%
%\vspace{10pt}
%\cventry{}{Funded by the American Department on Defense, in collaboration with Mayo Clinic (Dr. Wang, Dr. Weinshilboum, Dr. Tang and Prof. Kalari)}{Reverse Engineering the Human Cancer Transcriptome with Applications to Prostate Cancer}{}{}{} 
%\begin{itemize}
%\item A method to elucidate mechanistic changes in gene regulatory networks based on drug response.
%\item Applications to patients with metastatic castrate-resistant prostate cancer and their response to Abiraterone Acetate/Prednisone.
%\item Funded by the US Department of Defense
%\end{itemize}
%
%
%\vspace{10pt}
%\cventry{}{Funded by the Spanish National Reserach Plan}{Reducing the footprint of Genomic Information}{}{}{} 
%\begin{itemize}
%\item Develop novel technologies to reduce the storage and transmission cost of next-generation sequencing data.
%\item Previously funded by the Mayo Clinic Grand Challenge project.
%\end{itemize}
%
%\vspace{10pt}
%\cventry{}{under the Mitogen initiative (mitogen.github.io)}{MPEG-G, a new international standard for genomic information representation}{}{}{} 
%\begin{itemize}
%\item Work under ISO (International Organization for Standardization) to develop a new compressed standard for genomic information representation.
%\item Funded by NIH and the Chan-Zuckerberg Initiative under the Human Cell Atlas.
%\end{itemize}
%
%
%\vspace{10pt}
%\cventry{}{in collaboration with Stanford University and The Broad Institute (Prof. Gevaert and Prof. Pochet)}{Module-based analysis of the cancer transcriptome}{}{}{} 
%\begin{itemize}
%\item A computational method for multi-omics data fusion to identify cancer driver genes.
%\end{itemize}





\section{Education}
% \cventry{year--year}{Degree}{Institution}{City}{\textit{Grade}}{Description}



\cventry{09/2013--12/2016}{Postdoctoral researcher in the Dept. of Electrical Engineering}{\textsc{Stanford University}}{CA, USA}{}{}  % arguments 3 to 6 can be left empty
%\cventry{}{PhD candidate in Electrical Engineering}{\textsc{Stanford University, Stanford, CA, USA}}{09/2010--Present}{}{}  % arguments 3 to 6 can be left empty

\begin{itemize}
% \item Developed compression tools for genomic data
 %\item Explored new approaches for performing alignment and variant calling on genomic data
 %\item Characterized long non-coding RNAs across human cancer using unsupervised learning
  \item P.I.: Prof. Tsachy Weissman (tsachy@stanford.edu)
 % \item Collaborated with: 
  %\begin{itemize}
 % \item Prof. Euan Ashley, M.D. (euan@stanford.edu), Stanford Medical School
%  \item Prof. Olgica Milencovic (milenkov@illinois.edu), Electrical and Computer Eng. Dept.
 % \item Prof. Olivier Gevaert (ogevaert@stanford.edu), Stanford Data Science in Biomedicine Dept.
  %\end{itemize}
\end{itemize}

%\begin{itemize}
%\item Advisor: Prof. Tsachy Weissman (tsachy@stanford.edu)
%\item Thesis: Genomic data compression and processing: theory, models, algorithms and experiments.
%\end{itemize}

\vspace{10pt}
\cventry{09/2010--12/2012}{PhD in Electrical Engineering}{TECNUN, University of Navarra}{SPAIN}{\textit{GPA: Summa Cum Laude}}{}
% \cventry{09/2010--12/2012}{Master in Electrical Engineering}{Stanford University, Stanford, CA, USA}{}{}{GPA: 4.031}
\begin{itemize}
 %\item{Topics:} Conducted research in information theory and coding, communication systems and signal processing, with special focus on iterative channel codes (LDPC, turbo codes...)
 \item{Dissertation}: {Joint Network-Channel Coding Schemes for Relay Networks}
 \begin{itemize}
  \item Advisors: Prof. Pedro Crespo (pcrespo@tecnun.es) and Javier Del Ser (jdelser@tecnalia.es)
\end{itemize}
\end{itemize}

\vspace{10pt}
\cventry{Summer 2012}{Visiting Researcher}{Stanford University}{CA, USA}{}{}
%\begin{itemize}
%\item Supervisor: Golan Yona (golan.yona@stanford.edu)
%\item Conducted research in biological relational databases under the BIOZON project (http://biozon.org)
%\end{itemize}


\vspace{10pt}
\cventry{09/2003--02/2009}{Joint BSc/MSc in Telecommunications Engineering, Rank: Top 10 }{TECNUN, University of Navarra}{SPAIN}{}{}
\begin{itemize}
 \item{Master thesis:} {Concatenated LDGM Codes for the Transmission of Correlated Sources over Gaussian Broadcast Channels}  \emph{(GPA: 10/10)}
 \begin{itemize}
  %\item \href{http://web.stanford.edu/~iochoa/pfc_idoia.pdf}{Iterative Decoding Techniques for the Relay Channel}
  \item Advisor: Prof. Pedro M. Crespo (pcrespo@tecnun.es)
\end{itemize}
\end{itemize}
\vspace{10pt}

\cventry{08/2007--01/2008}{Erasmus Program}{Lulea Tekniska Universitet}{SWEDEN}{}{}



% \subsection{Vocational}


%\section{Interests}

%Data Compression, Bioinformatics, Machine learning, Information Theory and Coding, Signal Processing.

%Research interests:
%High-dimensional statistics, convex optimization, machine learning, applied probability and harmonic analysis. 
%Applications in signal processing, computer vision, medical imaging and big data.

\section{Past Work Experience}
% \subsection{Industry}
% \cventry{year--year}{Degree}{Institution}{City}{\textit{Grade}}{Description}


\cventry{2017--2019}{Director of Computational Genomics}{Carl R. Woese Institute for Genomic Biology, University of Illinois}{IL, USA}{}{Executive Director, CompGen Initiative}
%\begin{itemize}
%\item Advice leadership on the computational biology matters.
%\item Co-PI of the Mayo Grand Challenge project: Al collaboration between Mayo Clinic and the University of Illinois. Lead PI of the genomic data compression front.
%\item Develop statistical methods for RNA-Seq data in collaboration with the Carle College of Medicine and Stanford University.\\
%\end{itemize}
\vspace{10pt}

\cventry{2013--2016}{Postdoctoral Researcher}{Stanford University}{CA, USA}{}{}
%
%I worked on the design and development of new algorithms to improve the distribution and storage of genomic data, to facilitate its access, and to boost the inferential power of analysis performed on it. My approach combines tools from information theory, statistics, and machine learning.
%
\begin{itemize}
\item {Group of Prof. Tsachy Weissman (tsachy@stanford.edu)}
%\begin{itemize}
%\item Proposed, in collaboration with MIT and EPFL, a methodology for analysis of genomic data compression on Variant Calling that set the bases for the Standardization process of genomic information
%\item Designed lossless and lossy compressors for genomic data
%\item Developed a denoiser to reduce noise present in genomic data \\
%\end{itemize}
\end{itemize}
% Developed a framework to super-resolve point sources from bandlimited data via convex programming. Established non-asymptotic guarantees for the performance of the algorithm in a noisy setting, quantifying its support-detection accuracy and characterizing the quality of the estimate evaluated at a higher resolution.
\vspace{10pt}

\cventry{03/2013 - 09/2013}{Director of Research}{ENIGMEDIA}{SPAIN-USA}{}{}
\begin{itemize}
\item Supervisor: CEO \& Founder Gerard Vidal (gerard@enigmedia.com)
%\item Worked on encrypted real-time communications based on chaos-based stream-ciphers
\end{itemize}
\vspace{10pt}

%\cventry{04/2013 - 06/2013}{Consultant}{NAUI SYSTEMS}{CA, USA}{}{}
%\begin{itemize}
%\item Helped with the possibility of implementing a new coding solution for RAM memories
%\end{itemize}
%\vspace{10pt}



% \itemsep -2pt

%\cventry{2009--2012}{Research Assistant in the Electrical Engineering Department}{CEIT}{Spain}{}{}
%\begin{itemize}
%\item Supervisor: Prof. Pedro M. Crespo (pcrespo@ceit.es)
%\item Proposed several practical coding schemes for relay channels using LDPC and Turbo codes
%\item  Set up of a point-to-point wireless communication system for pedagogic purposes
%\end{itemize}
%\vspace{10pt}
%
%\cventry{2005}{Collaborator Student in the Electrical Engineering Department}{TECNUN, University of Navarra}{Spain}{}{}
%\begin{itemize}
%\item Worked on Cadence circuit design and lay-out
%\end{itemize}
%\vspace{10pt}

\section{Teaching experience}
\cventry{01/2020 -  }{Adjunct Professor}{University of Navarra}{SPAIN}{}{}
\begin{itemize}
\item Masters in Data Science  (School of Sciences)
\begin{itemize}
\item Master's committee member
\item Advance Topics in Machine Learning
\item Deep Learning
\end{itemize}
\end{itemize}
\vspace{10pt}

\cventry{01/2018 - 12/2019 }{Lecturer}{University of Illinois at Urbana-Champaign}{CA, USA}{}{}
\begin{itemize}
\item BIOE 383: High-Throughput Genomic Data Analysis (Dept. Bioengineering)
\item ECE 365: Data Science and Engineering (Dept. Electrical and Computer Engineering)
\end{itemize}
\vspace{10pt}

\cventry{09/2012 -  01/2013}{Lecturer}{TECNUN School of Engineering, University of Navarra}{CA, USA}{}{}
\begin{itemize}
\item Information Theory and Coding
\item Communication Systems
\item Fundamentals of Computers course
\end{itemize}
\vspace{10pt}

\cventry{2011--2012}{Teaching Assistant}{TECNUN School of Engineering, University of Navarra}{SPAIN}{}{}
\begin{itemize}
\item Information Theory and Coding
\item Communication Systems
\begin{itemize}
\item Set up of a point-to-point wireless communication system for pedagogic purposes
\end{itemize}
\end{itemize}
\vspace{10pt}

%\cventry{2010--2011}{Advisor of a Master Thesis}{TECNUN School of Engineering, University of Navarra}{Spain}{}{}
%\begin{itemize}
%\item {Topic:} Implementation of a Software Development Kit for Communications System.
%\item Resulted in a IEEE publication.
%\end{itemize}
%\vspace{10pt}

\section{Research Grants}
\cventry{220,000€}{Spanish Ministry of Science and Innovation}{Ramon y Cajal Fellowship (National Career award)}{2022-2027}{PI}{\textit{Machine Learning methods for translational biomedicine.}}
\vspace{10pt}
%
\cventry{\$1,500,000}{NIH (USA) / ISCIII (Spain)}{CRCNS US-Spain Research Proposal}{2022-2027}{co-PI}{\textit{Deep Learning Methods to Discover
Molecular Determinants of Neurovascular Coupling in Health and Disease.}}
\vspace{10pt}
%
\cventry{180,000€}{Spanish Ministry of Science and Innovation}{Spanish National Research Project}{2024-2027}{PI}{\textit{Explainable Deep Learning for tailored therapeutics to overcome drug resistance in cancer.}}
\vspace{10pt}
%
\cventry{12,000€}{Spanish Ministry of Education}{Jose Castillejo Mobility grant}{2025}{PI}{\textit{Research stay at New York University (NYU).}}
\vspace{10pt}
%
\cventry{1,000,000€}{La Caixa Foundation}{La Caixa Health Research grant}{2025-2027}{Senior Researcher}{\textit{Uncovering resistance mechanism to CAR-T cells via deep learning in scRNA-seq for improved therapies.}}
\vspace{10pt}
%
\cventry{187,000€}{Spanish Ministry of Science and Innovation}{Spanish Digital and Ecological Transition grant}{2022-2025}{co-PI}{\textit{Non-invasive cancer diagnosis using machine learning based characterization of circulating tumor cells.}}
\vspace{10pt}
%
\cventry{\$750,000}{US Department of Defense}{Congressionally-directed medical research program (CDMRP) grant}{2020-2024}{co-PI}{\textit{Novel Methods to Elucidate Abiraterone Resistance Mechanisms Using RNASeq Data and Xenograft Models from CRPC Patients.}}
\vspace{10pt}
%
\cventry{39,680 €}{Spanish Ministry of Science and Innovation}{Spanish National Research Project}{2021-2024}{PI}{\textit{Tools and mechanisms for secure compression and management of genomic information for clinical laboratories: Translational aspects.}}
\vspace{10pt}
%
\cventry{175,000€}{European Research Council}{Marie S. Curie - Individual Fellowships}{2020-2022}{PI}{\textit{Elucidating Transcriptional Rewiring on Hematological Malignancies via Computational Methods.}}
\vspace{10pt}
%
\cventry{\$105,000}{SVSF (Chan-Zuckerberg Initiative)}{CZI single-cell grant}{2018-2019}{PI}{\textit{Quantization and Compressive Learning Methods for Omics Data.}}
\vspace{10pt}
%
\cventry{\$150,000}{University of Illinois}{College of Engineering Intramural grant}{2018-2019}{co-PI}{\textit{Bringing digital era formats to genomic information. }}
\vspace{10pt}

%\section{Scientific Journals}

%\section{Preprints and under review}
%$^{\ast}$: Contacting author and/or (joint) first author\\

%\item C. Alberti, T. Paridaens, J. Voges, D. Naro, J. J. Ahmad, M. Ravasi, D. Renzi, P. Ribeca, G. Zoia, I. Ochoa, M. Mattavelli, J. Delgado,  \textbf{M. Hernaez}$^\ast$, \href{https://www.biorxiv.org/content/early/2018/10/08/426353}{\textsl{An introduction to MPEG-G, the new ISO standard for genomic information representation}}, Biorxiv, 2018.\\

%\item W. Arif, C. Blatti, A. Kalsotra, \textbf{M. Hernaez}$^\ast$, Shirloc: robust identification of ribosomal shifts from polysome profiling RNA-seq data, submitted to Nucleic Acid Research, 2019.\\

%\item X. Wang, Q. Li, Z. Ye, Y. Zhang, C. Blatti, \textbf{M. Hernaez}, and J. Han, EvidenceMiner: Text Evidence Discovery for Scientific Hypothesis Validation, submitted to Cell Systems.\\

\section{Journal Papers and Conference Proceedings }
$^{\ast}$: Contacting author and/or (joint) first author\\

\subsection{2025}
\vspace{6pt}

\begin{itemize}

%%%%% 2025 %%%%%%%%%

\item de la Fuente J, Serrano G, Veleiro U, Casals M, Vera L, Pizurica M, Gómez-Cebrián N, Puchades-Carrasco L, Pineda-Lucena A, Ochoa I, Vicent S, Gevaert O$^{\ast}$ and \textbf{Hernaez M}$^{\ast}$, \textsl{Towards an inductive world for drug repurposing approaches}. \textbf{Nature Machine Intelligence}, 2025. \\
In press\\


%%%%% 2024 %%%%%%%%%%=
\subsection{2024}
\vspace{6pt}

\item Goñi E, Mas AM, Gonzalez J, Abad A, Santisteban M, Fortes P, Huarte M$^{\ast}$, \textbf{Hernaez M}$^{\ast}$, \textsl{Uncovering functional lncRNAs by scRNA-seq with ELATUS}, \textbf{Nature Communications}, 2024.\\
doi: 10.1038/s41467-024-54005-7\\

\item Serrano G, Berastegui N, Díaz-Mazkiaran A, García-Olloqui P, Rodriguez-Res C, Huerga-Dominguez S, Ainciburu M, Vilas-Zornoza A, Martin-Uriz PS, Aguirre-Ruiz P, Ullate-Agote A, Ariceta B, Lamo-Espinosa JM, Acha P, Calvete O, Jimenez T, Molero A, Montoro MJ, Díez-Campelo M, Valcarcel D, Solé F, Alfonso-Pierola A, Ochoa I, Prosper F$^{\ast}$, Ezponda T$^{\ast}$, \textbf{Hernaez M}$^{\ast}$, \textsl{Single-cell transcriptional profile of CD34+ hematopoietic progenitor cells from del (5q) myelodysplastic syndromes and impact of lenalidomide}, \textbf{Nature Communications}, 2024.\\
doi: 10.1038/s41467-024-49529-x\\

\item Ruiz-Arenas C, Marin I, Wang L, Ochoa I, Perez-Jurado JL, \textbf{Hernaez M}$^{\ast}$, \textsl{NetActivity enhances transcriptional signals by combining gene expression into robust gene set activity scores through interpretable autoencoders}, \textbf{Nucleic Acids Research}, 2024.\\
doi: 10.1093/nar/gkae197\\

\item Veleiro U, de la Fuente J, Serrano G, Pizurica M, Casals M, Pineda-Lucena A, Vicent S, Ochoa I, Gevaert G$^{\ast}$, \textbf{Hernaez M}$^{\ast}$, \textsl{GeNNius: An ultrafast drug-target interaction inference method based on graph neural networks}, \textbf{Bioinformatics}, 2024.\\
doi: 10.1093/bioinformatics/btad774\\

\item Munterferring F, Chandak S, Adhisantoso Y, Ostermann J, \textbf{Hernaez M}$^{\ast}$, Voges J$^{\ast}$, \textsl{Genie: The First Open-Source ISO/IEC Encoder for Genomic Data}, \textbf{Communications Biology}, 2024.\\ 
doi: 10.1038/s42003-024-06249-8\\

\item Palacios-Berraquero ML, Rodriguez-Marquez P, Calleja-Cervantes ME, Berastegui N, Zabaleta A, Burgos L, Alignani D, San Martin-Uriz P, Vilas-Zornoza A, Rodriguez-Diaz S, Inoges S, Lopez-Diaz de Cerio A, Huerga S, Tamariz E, Rifon J, Alfonso-Pierola A, Lasarte JJ, Paiva B, \textbf{Hernaez M}, Rodriguez-Otero P, San-Miguel J, Ezponda T$^{\ast}$,  Rodriguez-Madoz JR$^{\ast}$, Prosper F$^{\ast}$. \textsl{Molecular mechanisms promoting long-term cytopenia after BCMA CAR-T therapy in Multiple Myeloma}, \textbf{Blood Advances}, 2024.\\
doi: 10.1182/bloodadvances.2023012522\\

\item de la Nava D, Ausejo-Mauleon I, Laspidea V, Gonzalez-Huarriz M, Lacalle A, Casares N, Zalacain M, Marrodan L, García-Moure M, Ochoa MC, Tallon-Cobos AC, Hernandez-Osuna R, Marco-Sanz J, Dhandapani L, Hervás-Corpión I, Becher OJ, Nazarian J, Mueller S., Phoenix TN, van der Lugt J, \textbf{Hernaez M}, Guruceaga E, Koschmann C, Venneti S, Allen JE, Dun MD, Fueyo J, Gomez-Manzano C, Gallego Perez-Larraya J, Patiño-García A, Labiano S, Alonso MM$^{\ast}$, \textsl{The oncolytic adenovirus Delta-24-RGD in combination with ONC201 induces a potent antitumor response in pediatric high-grade and diffuse midline glioma models}, \textbf{Neuro-oncology}, 2024.\\
doi: 10.1093/neuonc/noae066\\

\item Patiño-García A$^{\ast}$, Guruceaga E, Andueza MP, Ocón M, Fodop Sokoudjou JJ, de Villalonga Zornoza N, Alkorta-Aranburu G, Tamayo I, Gurpide A, Camps C, Jantus-Lewintre E, Navamuel-Andueza M, Sanmamed MF, Melero I, Elgendy M, Fusco JP, Zulueta JJ, de-Torres JP, Bastarrika G, Seijo L, Pio R, Montuenga LM, \textbf{Hernaez M}, Ochoa I, Perez-Gracia JL$^{\ast}$, \textsl{Whole exome sequencing and machine learning germline analysis of individuals presenting with extreme phenotypes of high and low risk of developing tobacco-associated lung adenocarcinoma}, \textbf{EBioMedicine}, 2024.\\
doi: 10.1016/j.ebiom.2024.105048\\

\item Barace S, Santamaría E, Infante S, Arcelus S, De La Fuente J, Goñi E, Tamayo I, Ochoa I, Sogbe M, Sangro B, \textbf{Hernaez M}, Avila MA$^{\ast}$, Argemi J$^{\ast}$, \textsl{Application of Graph Models to the Identification of Transcriptomic Oncometabolic Pathways in Human Hepatocellular Carcinoma. Biomolecules}, 2024. \\
doi: 10.3390/biom14060653. \\

\item Naro D, Delgado J, Llorente S, \textbf{Hernaez M}, \textsl{Use of Beacon v2 for Improving Genomics Based Research in a Clinical Setting}, \textbf{Studies in Health Technology and Informatics}, 2024.\\
doi: 10.3233/SHTI240636\\

\item Torella L, Klermund J, Bilbao-Arribas M, Tamayo I, Andrieux G, Chmielewski KO, Vales A., Olagüe C, Moreno-Luqui D, Raimondi I, Abad A, Torrens-Baile J, Salido E, Huarte M, \textbf{Hernaez M}, Boerries M, Cathomen T, Zabaleta N, Gonzalez-Aseguinolaza G$^{\ast}$, \textsl{Efficient and Safe Therapeutic Use of Paired Cas9-Nickases for Primary Hyperoxaluria Type 1},  \textbf{EMBO Molecular Medicine}, 2024.\\
doi: 10.1038/s44321-023-00008-8\\

%%%%%%%%% 2023 %%%%%%%%%%%%%%
\subsection{2023}
\vspace{6pt}

\item Blatti C, de la Fuente J, Gao H, Marín-Goñi I, Chen Z, Zhao SD, Tan W, Weinshilboum R, Kalari KR, Wang L$^{\ast}$, \textbf{Hernaez M}$^{\ast}$, \textsl{Bayesian Machine Learning Enables Identification of Transcriptional Network Disruptions Associated with Drug-Resistant Prostate Cancer}. \textbf{Cancer Research}, 2023.\\
doi: 10.1158/0008-5472.CAN-22-1910.\\

\item Bakr S, Brennan K, Mukherjee P, Argemi J, \textbf{Hernaez M}$^{\ast}$, Gevaert O$^{\ast}$, \textsl{Identifying key multifunctional components shared by critical cancer and normal liver pathways via SparseGMM}, \textbf{Cell Reports Methods}, 2023\\
doi: 10.1016/j.crmeth.2022.100392.\\

\item Calviño C, Ceballos C, Alfonso A, Jauregui P, Calleja-Cervantes ME, San Martin-Uriz P, Rodriguez-Marquez P, Martin-Mallo A, Iglesias E, Abizanda G, Rodriguez-Diaz S, Martinez-Turrillas R, Illarramendi J, Viguria MC, Redondo M, Rifon J, Villar S, Lasarte JJ, Inoges S, Lopez-Diaz de Cerio A, \textbf{Hernaez M}$^{\ast}$, Prosper F$^{\ast}$, Rodriguez-Madoz JR$^{\ast}$, \textsl{Optimization of universal allogeneic CAR-T cells combining CRISPR and transposon-based technologies for treatment of acute myeloid leukemia}. \textbf{Frontiers in Immunology}, 2023.\\
doi: 10.3389/fimmu.2023.1270843.\\

\item Traniello IM, Bukhari SA, Dibaeinia P, Serrano G, Avalos A, Ahmed AC, Sankey AL, \textbf{Hernaez M}, Sinha S, Zhao SD, Catchen J, Robinson GE$^{\ast}$. \textsl{Single-cell dissection of aggression in honeybee colonies}, \textbf{Nature Ecology and Evolution}, 2023.\\ 
doi: 10.1038/s41559-023-02090-0.\\

\item Ainciburu M, Ezponda T, Berastegui N, Alfonso-Pierola A, Vilas-Zornoza A, San Martin-Uriz P, Alignani D, Lamo-Espinosa J, San-Julian M, Jiménez-Solas T, Lopez F, Muntion S, Sanchez-Guijo F, Molero A, Montoro J, Serrano G, Diaz-Mazkiaran A, Lasaga M, Gomez-Cabrero D, Diez-Campelo M, Valcarcel D, \textbf{Hernaez M}, Romero JP$^{\ast}$, Prosper F$^{\ast}$. \textsl{Uncovering perturbations in human hematopoiesis associated with healthy aging and myeloid malignancies at single-cell resolution}, \textbf{Elife}, 2023.\\
doi: 10.7554/eLife.79363.\\

\item Basurco L, Abellanas MA, Ayerra L, Conde E, Vinueza-Gavilanes R, Luquin E, Vales A, Vilas A, Martin-Uriz PS, Tamayo I, Alonso MM, \textbf{Hernaez M}, Gonzalez-Aseguinolaza G, Clavero P, Mengual E, Arrasate M, Hervás-Stubbs S, Aymerich MS$^{\ast}$, \textsl{Microglia and astrocyte activation is region-dependent in the $\alpha$-synuclein mouse model of Parkinson's disease}, \textbf{Glia}, 2023.\\
doi: 10.1002/glia.24295.\\


%%%%%%%%% 2022 %%%%%%%%%%%%%%
\subsection{2022}
\vspace{6pt}

\item Rodriguez-Marquez P, Calleja-Cervantes ME, Serrano G, Oliver-Caldes A, Palacios-Berraquero ML, Martin-Mallo A, Calviño C, Español-Rego M, Ceballos C, Lozano T, San Martin-Uriz P, Vilas-Zornoza A, Rodriguez-Diaz S, Martinez-Turrillas R, Jauregui P, Alignani D, Viguria MC, Redondo M, Pascal M, Martin-Antonio B, Juan M, Urbano-Ispizua A, Rodriguez-Otero P, Alfonso-Pierola A, Paiva B, Lasarte JJ, Inoges S, Lopez-Diaz de Cerio A, San-Miguel J, Fernandez de Larrea C, \textbf{Hernaez M}$^{\ast}$, Rodriguez-Madoz JR$^{\ast}$, Prosper F$^{\ast}$. CAR density influences antitumoral efficacy of BCMA CAR T cells and correlates with clinical outcome. \textbf{Science Advances}. 2022.\\
doi: 10.1126/sciadv.abo0514.\\

\item Goyal M, Serrano G, Argemi J, Shomorony I, \textbf{Hernaez M}$^{\ast}$, Ochoa I$^{\ast}$, \textsl{JIND: joint integration and discrimination for automated single-cell annotation}, \textbf{Bioinformatics}. 2022\\ 
doi: 10.1093/bioinformatics/btac140.\\

\item Peng J, Serrano G, Traniello IM, Calleja-Cervantes ME, Chembazhi UV, Bangru S, Ezponda T, Rodriguez-Madoz JR, Kalsotra A, Prosper F, Ochoa I$^{\ast}$, \textbf{Hernaez M}$^{\ast}$, \textsl{SimiC enables the inference of complex gene regulatory dynamics across cell phenotypes}. \textbf{Communications Biology}. 2022\\
doi: 10.1038/s42003-022-03319-7.\\

\item Berastegui N, Ainciburu M, Romero JP, Garcia-Olloqui P, Alfonso-Pierola A, Philippe C, Vilas-Zornoza A, San Martin-Uriz P, Ruiz-Hernández R, Abarrategi A, Ordoñez R, Alignani D, Sarvide S, Castro-Labrador L, Lamo-Espinosa JM, San-Julian M, Jimenez T, López-Cadenas F, Muntion S, Sanchez-Guijo F, Molero A, Montoro MJ, Tazón B, Serrano G, Diaz-Mazkiaran A, \textbf{Hernaez M}, Huerga S, Bewicke-Copley F, Rio-Machin A, Maurano MT, Díez-Campelo M, Valcarcel D, Rouault-Pierre K, Lara-Astiaso D, Ezponda T$^{\ast}$, Prosper F$^{\ast}$, \textsl{The transcription factor DDIT3 is a potential driver of dyserythropoiesis in myelodysplastic syndromes}. \textbf{Nature Communications}, 2022.\\ 
doi: 10.1038/s41467-022-35192-7.\\

%%%%%%%%% 2021 %%%%%%%%%%%%%%
\subsection{2021}
\vspace{6pt}

\item \textbf{Hernaez M}$^{\ast}$. \textsl{Towards scalable genomic data access}. \textbf{Nature Computational Science}. 2021\\
doi: 10.1038/s43588-021-00089-w.\\

\item Voges J, \textbf{Hernaez M}, Mattavelli M, Ostermann J$^{\ast}$. \textsl{An introduction to mpeg-g: the first open iso/iec standard for the compression and exchange of genomic sequencing data}. \textbf{Proceedings of the IEEE}. 2021.\\
doi: 10.1109/JPROC.2021.3082027\\

\item Chembazhi UV, Bangru S, \textbf{Hernaez M}, Kalsotra A$^{\ast}$. \textsl{Cellular plasticity balances the metabolic and proliferation dynamics of a regenerating liver}. \textbf{Genome Research}. 2021\\
doi: 10.1101/gr.267013.120.\\

%%%%%%%%% 2020 %%%%%%%%%%%%%%
\subsection{2020}
\vspace{6pt}

\item \textbf{Hernaez M}$^\ast$, Blatti C, Gevaert O, \textsl{Comparison of single gene and module-based methods for modeling gene regulatory networks}, \textbf{Bioinformatics}, 2020.\\
doi: 10.1093/bioinformatics/btz549\\

\item Meng Q, Ochoa I, \textbf{Hernaez M}$^\ast$, \textsl{GPress: a framework for querying General Feature Format (GFF) files and expression files in a compressed form}, \textbf{Bioinformatics}, 2020.\\
doi: 10.1093/bioinformatics/btaa604\\

\item Voges J, Paridaens T, Müntefering F, Mainzer LS, Bliss B, Yang M, Ochoa I, Fostier J, Ostermann J$^\ast$, \textbf{Hernaez M}$^\ast$. \textsl{GABAC: an arithmetic coding solution for genomic data}. \textbf{Bioinformatics}. 2020\\ 
doi: 10.1093/bioinformatics/btz922.\\

\item No A, \textbf{Hernaez M}$^{\ast}$, Ochoa I$^{\ast}$. \textsl{CROMqs: An infinitesimal successive refinement lossy compressor for the quality scores}. \textbf{Journal of Bioinformatics and Computational Biology}. 2020\\
doi: 10.1142/S0219720020500316.\\

\item Gevaert O$^{\ast}$, Nabian M, Bakr S, Everaert C, Shinde J, Manukyan A, Liefeld T, Tabor T, Xu J, Lupberger J, Haas BJ, Baumert TF, \textbf{Hernaez M}, Reich M, Quintana FJ, Uhlmann EJ, Krichevsky AM, Mesirov JP, Carey V, Pochet N$^{\ast}$. \textsl{Imaging-AMARETTO: An Imaging Genomics Software Tool to Interrogate Multiomics Networks for Relevance to Radiography and Histopathology Imaging Biomarkers of Clinical Outcomes}. \textbf{JCO Clinical Cancer Informatics}. 2020\\ 
doi: 10.1200/CCI.19.00125.\\ 

%%%%%%%%% 2019 %%%%%%%%%%%%%%
\subsection{2019}
\vspace{6pt}

\item Chandak S$^\ast$, Ochoa I,  \textbf{Hernaez M}$^\ast$, Weissman T, \textsl{SPRING: a next-generation compressor for FASTQ data}, \textbf{Bioinformatics}, 2019.\\
doi: 10.1093/bioinformatics/bty1015.\\

\item \textbf{Hernaez M}$^\ast$, Pavlichin D, and Weissman T, Ochoa I$^\ast$, \textsl{Genomic Data Compression}, \textbf{Annual Review of Biomedical Data Science}, 2019. \\
doi: 10.1146/annurev-biodatasci-072018-021229\\

\item Fisher-Hwang I, Ochoa I,  Weissman T, \textbf{Hernaez M}$^\ast$, \textsl{Denoising of Aligned Genomic Data}, \textbf{Scientific Reports}, 2019.\\
doi: 10.1038/s41598-019-51418-z\\

\item Ochoa I, Li H, Baumgarte F, Hergenrother C, Voges J, \textbf{Hernaez M}, {\emph{AliCo: A New Efficient Representation for SAM Files}}, {\textbf{Proceedings of the Data Compression Conference (DCC)}}, 2019. \\
doi: 10.1109/DCC.2019.00017.\\

\item Zheng H, Brennan K, \textbf{Hernaez M}, Gevaert O$^\ast$, \textsl{Benchmark of lncRNA Quantification for RNA-Seq of Cancer Samples}, \textbf{GigaScience}, 2019.\\
doi: 10.1093/gigascience/giz145. \\

\item Kendig KI, Baheti S, Bockol MA, Drucker TM, Hart SN, Heldenbrand JR, \textbf{Hernaez M}, Hudson ME, Kalmbach MT, Klee EW, Mattson NR, Ross CA, Taschuk M, Wieben ED, Wiepert M, Wildman DE, Mainzer LS$^\ast$. \textsl{Sentieon DNASeq Variant Calling Workflow Demonstrates Strong Computational Performance and Accuracy}. \textbf{Frontiers in Genetics}. 2019\\
doi: 10.3389/fgene.2019.00736.\\

\item Seimetz J, Arif W, Bangru B, \textbf{Hernaez M}, Kalsotra A, \textsl{Cell-type specific polysome profiling from mammalian tissues}, \textbf{Methods}, 155, 2019.\\
doi: 10.1016/j.ymeth.2018.11.015\\


%%%%%%%%% 2018 %%%%%%%%%%%%%%
\subsection{2018}
\vspace{6pt}

\item Rogusky L, Ochoa I, \textbf{Hernaez M}, Deorowicz S$^\ast$, \textsl{FaStore - a space-saving solution for raw sequencing data}, \textbf{Bioinformatics}, 2018.\\
doi: 10.1093/bioinformatics/bty205\\

\item Voges J, Oesterman J, \textbf{Hernaez M}$^\ast$, \textsl{CALQ: compression of quality values of aligned sequencing data}, \textbf{Bioinformatics}, 2018.\\
doi: 10.1093/bioinformatics/btx737\\


%%%%%%%%% 2017 %%%%%%%%%%%%%%
\subsection{2017}
\vspace{6pt}

\item Long R, \textbf{Hernaez M}, I Ochoa, T Weissman, \textsl{GeneComp, a new reference-based compressor for SAM files}, \textbf{Proceedings of the Data Compression Conference (DCC)}, 2017.\\
doi: 10.1109/DCC.2017.76.\\

\item Ochoa I$^\ast$, \textbf{Hernaez M}, Goldfeder R, Weissman T, Ashley E. \textsl{Effect of lossy compression of quality scores on variant calling}. \textbf{Briefings in Bioinformatics}. 2017. \\
doi: 10.1093/bib/bbw011.\\


%%%%%%%%% 2016 %%%%%%%%%%%%%%
\subsection{2016}
\vspace{6pt}

\item Tatwawadi T.$^\ast$, \textbf{Hernaez M}, Ochoa I, and Weissman T, \textsl{GTRAC: fast retrieval from compressed collections of genomic variants}, \textbf{Bioinformatics}, 2016.\\
doi: 10.1093/bioinformatics/btw437.\\

\item Alberti C, Daniels , \textbf{Hernaez M}, Voges J, Goldfeder RL, Hernandez-Lopez AA, Mattavelli M, Berger B, \textsl{An Evaluation Framework for Lossy Compression of Genome Sequencing Quality Values},  \textbf{Proceedings of the Data Compression Conference (DCC)}, 2016. \\
doi: 10.1109/DCC.2016.39.\\

\item Ochoa I, \textbf{Hernaez M}, Goldfeder RL, Weissman T, Ashley E, \textsl{Denoising of Quality Scores for Boosted Inference and Reduced Storage}, \textbf{ Proceedings of the Data Compression Conference (DCC)}, 2016. \\
doi: 10.1109/DCC.2016.92.\\

\item  \textbf{Hernaez M}, Ochoa I, Weissman T, \textsl{A cluster-based approach to compression of Quality Scores}, \textbf{Proceedings of the Data Compression Conference (DCC)}, 2016. \\
doi: 10.1109/DCC.2016.49\\

\item Deorowicz S$^\ast$, Grabowski S, Ochoa I$^\ast$, \textbf{Hernaez M}, Weissman T. \textsl{Comment on: 'ERGC: an efficient referential genome compression algorithm'}. \textbf{Bioinformatics}. 2016.\\
doi: 10.1093/bioinformatics/btv704.\\


%%%%%%%%% 2015 %%%%%%%%%%%%%%
\subsection{2015}
\vspace{6pt}

\item Malysa G, \textbf{Hernaez M}$^\ast$, Ochoa I$^\ast$, Rao M, Ganesan K, Weissman T, \textsl{ QVZ: lossy compression of quality values}, \textbf{Bioinformatics}, 2015.\\
doi: 10.1093/bioinformatics/btv330.\\

\item Alustiza I, \textbf{Hernaez M}, Crespo P,\textsl{Design of a new scheme for multi-hop wireless networks using decode-and-forward strategy}, \textbf{EURASIP Journal on Wireless Communications and Networking}. 2015.\\
doi: 10.1186/s13638-015-0372-8.\\

%%%%%%%%% 2014 %%%%%%%%%%%%%%
\subsection{2014}
\vspace{6pt}

\item Ochoa I, \textbf{Hernaez M} and Weissman T, \textsl{Aligned genomic data compression via improved modeling}, \textbf{Journal of bioinformatics and computational biology}, 2014.\\
 doi: 10.1142/S0219720014420025.\\

\item Ochoa I$^\ast$, \textbf{Hernaez M} and T. Weissman, \textsl{ iDoComp: a compression scheme for assembled genomes}, \textbf{Bioinformatics}, 2014.\\
doi: 10.1093/bioinformatics/btu698\\

%%%%%%%%% 2013 %%%%%%%%%%%%%%
\subsection{2013}
\vspace{6pt}

\item \textbf{Hernaez M}, Crespo PM, {Del Ser} J, \textsl{ On the Design of a Novel Joint Network-Channel Coding Scheme for the Multiple Access Relay Channel}, \textbf{IEEE Journal on Selected Areas in Communications}, 2013.\\
doi: 10.1109/JSAC.2013.130802\\

\item \textbf{Hernaez M}, Crespo PM, {Del Ser} J, \textsl{A Decode-and-Forward Scheme for Multihop Wireless Networks}, \textbf{Proceedings of the IEEE Vehicular Technology Conference (VTC2013-Fall)}, 2013\\
doi: 10.1109/VTCFall.2013.6692030.\\

\item Alustiza I, \textbf{Hernaez M}, Insasusti X, Crespo PM, \textsl{Teaching Information Theory via a Simulation Tool for Communications Systems}, \textbf{IEEE Collaborative Learning \& New Pedagogic Approaches in Engineering Education (IEEE EDUCON)}, 2013.\\
doi: 10.1109/EduCon.2013.6530098\\

%%%%%%%%% 2012 %%%%%%%%%%%%%%
\subsection{2012}
\vspace{6pt}

\item \textbf{Hernaez M}, Crespo PM, {Del Ser} J, \textsl{A Flexible Channel Coding Approach for Short-Length Codewords}, \textbf{IEEE Communications Letters}, 2012.\\
doi: 10.1109/LCOMM.2012.073112.121295.\\

%%%%%%%%% 2011 %%%%%%%%%%%%%%
\subsection{2011}
\vspace{6pt}

\item \textbf{Hernaez M}, Crespo PM, \textsl{A novel Scheme for Message-Forwarding in Ad-Hoc Wireless Networks}, \textbf{Proceedings of the IEEE Vehicular Technology Conference (VTC2011-Spring)}, 2011.\\
doi: 10.1109/VETECS.2011.5956446.\\

\item \textbf{Hernaez M}, Crespo PM, {Del Ser} J, \textsl{Joint Non-Binary LDPC-BICM and Network Coding with Iterative Decoding for the Multiple Access Relay Channel}, \textbf{Proceedings of the IEEE Vehicular Technology Conference (VTC2011-Spring)}, 2011. \\
doi: 10.1109/VETECS.2011.5956367.\\

%%%%%%%%% 2010 %%%%%%%%%%%%%%
\subsection{2010}
\vspace{6pt}

\item Ochoa I, Crespo P, \textbf{Hernaez M}, \textsl{LDPC Codes for Non-Uniform Memoryless Sources and Unequal Energy Allocation}, \textbf{IEEE Communications Letters}, 2010.\\
doi: 10.1109/LCOMM.2010.09.100662.\\

\item Ochoa I, Crespo P, {Del Ser} J,  \textbf{Hernaez M}, \textsl{Turbo Joint Source-Channel Coding of Non-Uniform Memoryless Sources in the Bandwidth-Limited Regime}, \textbf{IEEE Communications Letters}, 2010.\\
doi: 10.1109/LCOMM.2010.04.092462\\

\item Ochoa I, Crespo PM, Del Ser J, \textbf{Hernaez M}. \textsl{Joint Turbo Coding and Source-Controlled Modulation of Cycle-Stationary Sources in the Bandwidth-Limited Regime.} \textbf{Mobile Lightweight Wireless Systems}, 2010.\\
doi: 10.1007/978-3-642-16644-0\_53.\\

%%%%%%%%% 2009 %%%%%%%%%%%%%%
\subsection{2009}
\vspace{6pt}

\item \textbf{Hernaez M}$^\ast$, Crespo PM, {Del Ser} J, Garcia-Frias J, \textsl{ Serially-Concatenated LDGM Codes for Correlated Sources over Gaussian Broadcast Channels}, \textbf{IEEE Communications Letters}, 2009.
doi: 10.1109/LCOMM.2009.091289\\


\end{itemize}

%\section{Conference Presentations}
%\section{Conference Papers - non proceedings}

%\begin{itemize}

%\item I. Ochoa, \textbf{M. Hernaez} and T. Weissman, \href{http://web.stanford.edu/~iochoa/publishedPublications/2014_cbc.pdf}{\emph{ Aligned genomic data compression via improved modeling}}, \textbf{GIW ISCB-Asia}, Japan, December 2014. \\

%\item \textbf{M. Hernaez}, G. Vidal, {\emph {Communication Services Empowered with a Classical Chaos Based Cryptosystem}, \textbf{Financial Cryptography 2013}}, Okinawa, Japan 2013 \\

%\item {I. Ochoa, P. Crespo, J. Del Ser and \textbf{M. Hernaez}, \href{http://web.stanford.edu/~iochoa/publishedPublications/2010_mobilight.pdf}{\emph{ Turbo Joint Source-Channel Coding of Cycle-Stationary Sources in the Bandwidth-Limited Regime}}, The 2nd International Conference on Mobile Lightweight Wireless Systems (MOBILIGHT), Spain, May 2010.}\\
 %Barcelona,
 
% \end{itemize}


\section{Patents}

% \newline
\cvlistitem{{Voges J, \textbf{Hernaez M}, Ostermann J, {\emph{Method for encoding and decoding of quality values of a data structure}}, US Patent App. 16/341,307, 2021.\\}}



\cvlistitem{{Chandak S, Tatwawadi K, Weissman T, Ochoa I, \textbf{Hernaez M}, {\emph{Systems and Methods for Compressing Genetic Sequencing Data and Uses Thereof}}, US Patent App. 16/545,751, 2020\\}}

\section{Scholarships and Awards}

\cvitemwithcomment{}{ISO/IEC Excellence Award}{2023}
\cvitemwithcomment{}{Ramon y Cajal Fellowship, Spanish Ministry of Science (Spanish National Career Award)}{2023}
\cvitemwithcomment{}{Marie S. Curie fellowship, ERC (European Pathway to Independence Award)}{2020}
\cvitemwithcomment{}{Stanford Data Science Initiative Postdoctoral Fellowship}{2015-2016}
\cvitemwithcomment{}{Enigmedia, inc. named “Best New Company of the Basque Country” (Spain)}{2013}
\cvitemwithcomment{}{University of Navarra Fellowship for PhD program}{2009--2011} 
\cvitemwithcomment{}{Telefonica Fellowship for Master's thesis (largest Teleco. operator in Spain)}{2008}


\section{Service Activities}
\cvitem{Workshop Organization Committees}{}
\begin{itemize}
\item Chair of special session on ``Omics Data Compression and Storage: Present and Future'' at ISMB/ECCB, Basel, 2019 (acceptance rate 15\%).
\item Chair of special session on ``Omics Data Compression and Storage: Present and Future'' at ISMB (International Society for Computational Biology), Chicago, 2018 (acceptance rate 20\%).
\item Chair of special session on ``Bioinformatics'' at the 56th Annual Allerton Conference on Communication, Control, and Computing (Allerton), October, 2018.
\item Chair of special session on ``Bioinformatics'' at the 55th Annual Allerton Conference on Communication, Control, and Computing (Allerton), October, 2017.
\end{itemize}


\cvitem{Professional Organizations}{}
\begin{itemize}
\item International Society of Computational Biology (ISCB): Member 
\item \href{https://compression.stanford.edu/}{Stanford Compression Forum}: Organizer of the first and second edition (2015 - 2016)
\item International Organization for Standardization (ISO): Active participant in the initiative to define and establish a compression standard for genomic data (under the MPEG working group).
\item \href{https://sdsi.stanford.edu/}{Stanford Data Science Initiative (SDSI)}: Active member and grantee (2014 - 2016)
\item \href{https://www.soihub.org/}{Center for Science of Information (CSoI), NSF Science and Technology Center}: Active member and grantee (2013-2015)
\end{itemize}
\vspace{5pt}



\section{Additional Information}
% \cvitem{International Organization for Standardization (ISO)}{Active participant in the initiative to define and stablish a compression standard for genomic data (under the MPEG working group).}
% \cvitem{}{Part of the \href{https://sdsi.stanford.edu/}{Stanford Data Science Initiative (SDSI)}}
\cvitem{Reviewer}{Bioinformatics, Nature Technical Reports, Nature Biotechnology, BMC Bioinformatics, IEEE Communications Letters, several conference proceedings.}
%\cvitem{Languages}{Native: Spanish, Proficiency: English, Low-Intermediate: German, French and Basque.}
%\cvitem{Computer skills}{Programming Languages: C/C++, Python, Applications: R, MatLab, LATEX, MS Office, CVX, Java Operating Systems: Linux, UNIX, Windows.}
%\cvitem{Student member}{Institute of Electrical and Electronics Engineers (IEEE)}

%\section{Experience}
%\subsection{Vocational}
%\cventry{year--year}{Job title}{Employer}{City}{}{General description no longer than 1--2 lines.\newline{}%
%Detailed achievements:%
%\begin{itemize}%
%\item Achievement 1;
%\item Achievement 2, with sub-achievements:
%  \begin{itemize}%
%  \item Sub-achievement (a);
%  \item Sub-achievement (b), with sub-sub-achievements (don't do this!);
%    \begin{itemize}
%    \item Sub-sub-achievement i;
%    \item Sub-sub-achievement ii;
%    \item Sub-sub-achievement iii;
%    \end{itemize}
%  \item Sub-achievement (c);
%  \end{itemize}
%\item Achievement 3.
%\end{itemize}}
%\cventry{year--year}{Job title}{Employer}{City}{}{Description line 1\newline{}Description line 2}
%\subsection{Miscellaneous}
%\cventry{year--year}{Job title}{Employer}{City}{}{Description}
%
%\section{Languages}
%\cvitemwithcomment{Language 1}{Skill level}{Comment}
%\cvitemwithcomment{Language 2}{Skill level}{Comment}
%\cvitemwithcomment{Language 3}{Skill level}{Comment}
%
%\section{Computer skills}
%\cvdoubleitem{category 1}{XXX, YYY, ZZZ}{category 4}{XXX, YYY, ZZZ}
%\cvdoubleitem{category 2}{XXX, YYY, ZZZ}{category 5}{XXX, YYY, ZZZ}
%\cvdoubleitem{category 3}{XXX, YYY, ZZZ}{category 6}{XXX, YYY, ZZZ}
%
%\section{Interests}
%\cvitem{hobby 1}{Description}
%\cvitem{hobby 2}{Description}
%\cvitem{hobby 3}{Description}
%
%\section{Extra 1}
%\cvlistitem{Item 1}
%\cvlistitem{Item 2}
%\cvlistitem{Item 3. This item is particularly long and therefore normally spans over several lines. Did you notice the indentation when the line wraps?}
%
%\section{Extra 2}
%\cvlistdoubleitem{Item 1}{Item 4}
%\cvlistdoubleitem{Item 2}{Item 5\cite{book1}}
%\cvlistdoubleitem{Item 3}{Item 6. Like item 3 in the single column list before, this item is particularly long to wrap over several lines.}
%
%\section{References}
%\begin{cvcolumns}
%  \cvcolumn{Category 1}{\begin{itemize}\item Person 1\item Person 2\item Person 3\end{itemize}}
%  \cvcolumn{Category 2}{Amongst others:\begin{itemize}\item Person 1, and\item Person 2\end{itemize}(more upon request)}
%  \cvcolumn[0.5]{All the rest \& some more}{\textit{That} person, and \textbf{those} also (all available upon request).}
%\end{cvcolumns}

% Publications from a BibTeX file without multibib
%  for numerical labels: \renewcommand{\bibliographyitemlabel}{\@biblabel{\arabic{enumiv}}}% CONSIDER MERGING WITH PREAMBLE PART
%  to redefine the heading string ("Publications"): \renewcommand{\refname}{Articles}
\nocite{*}
\bibliographystyle{plain}
\bibliography{publications}                        % 'publications' is the name of a BibTeX file

% Publications from a BibTeX file using the multibib package
%\section{Publications}
%\nocitebook{book1,book2}
%\bibliographystylebook{plain}
%\bibliographybook{publications}                   % 'publications' is the name of a BibTeX file
%\nocitemisc{misc1,misc2,misc3}
%\bibliographystylemisc{plain}
%\bibliographymisc{publications}                   % 'publications' is the name of a BibTeX file

%\clearpage
%%-----       letter       ---------------------------------------------------------
%% recipient data
%\recipient{Company Recruitment team}{Company, Inc.\\123 somestreet\\some city}
%\date{January 01, 1984}
%\opening{Dear Sir or Madam,}
%\closing{Yours faithfully,}
%\enclosure[Attached]{curriculum vit\ae{}}          % use an optional argument to use a string other than "Enclosure", or redefine \enclname
%\makelettertitle
%
%Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis ullamcorper neque sit amet lectus facilisis sed luctus nisl iaculis. Vivamus at neque arcu, sed tempor quam. Curabitur pharetra tincidunt tincidunt. Morbi volutpat feugiat mauris, quis tempor neque vehicula volutpat. Duis tristique justo vel massa fermentum accumsan. Mauris ante elit, feugiat vestibulum tempor eget, eleifend ac ipsum. Donec scelerisque lobortis ipsum eu vestibulum. Pellentesque vel massa at felis accumsan rhoncus.
%
%Suspendisse commodo, massa eu congue tincidunt, elit mauris pellentesque orci, cursus tempor odio nisl euismod augue. Aliquam adipiscing nibh ut odio sodales et pulvinar tortor laoreet. Mauris a accumsan ligula. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Suspendisse vulputate sem vehicula ipsum varius nec tempus dui dapibus. Phasellus et est urna, ut auctor erat. Sed tincidunt odio id odio aliquam mattis. Donec sapien nulla, feugiat eget adipiscing sit amet, lacinia ut dolor. Phasellus tincidunt, leo a fringilla consectetur, felis diam aliquam urna, vitae aliquet lectus orci nec velit. Vivamus dapibus varius blandit.
%
%Duis sit amet magna ante, at sodales diam. Aenean consectetur porta risus et sagittis. Ut interdum, enim varius pellentesque tincidunt, magna libero sodales tortor, ut fermentum nunc metus a ante. Vivamus odio leo, tincidunt eu luctus ut, sollicitudin sit amet metus. Nunc sed orci lectus. Ut sodales magna sed velit volutpat sit amet pulvinar diam venenatis.
%
%Albert Einstein discovered that $e=mc^2$ in 1905.
%
%\[ e=\lim_{n \to \infty} \left(1+\frac{1}{n}\right)^n \]
%
%\makeletterclosing

%\clearpage\end{CJK*}                              % if you are typesetting your resume in Chinese using CJK; the \clearpage is required for fancyhdr to work correctly with CJK, though it kills the page numbering by making \lastpage undefined
\end{document}


%% end of file `template.tex'.
